src/main/scala/decider/SmtlibNameGenerator.scala

```diff
@@ -128,6 +128,65 @@ class SmtlibNameSanitizer extends AbstractNameSanitizer {
     "re.inter", "seq.to.re", "seq.in.re", "re.all", "re.empty",
 
     /* Other reserved words (Z3 specific?) */
-    "min", "List", "const"
+    "min", "List", "const",
+
+    /* cvc5 - Transcendental operators, see https://github.com/cvc5/cvc5/blob/main/src/parser/smt2/smt2.cpp */
+    "exp", "sin", "cos", "tan", "csc", "sec", "cot", "arcsin", "arccos", "arctan", "arccsc", "arcsec", "arccot", "sqrt",
```

Comment:
How much of this is CVC5 specific? Are most of these also keywords to Z3? (I see some duplicates.)